core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -256,7 +307,29 @@ get_database_sort(Db1, Db2) ->
 
 -spec refresh(kz_term:ne_binary()) -> 'ok'.
 refresh(Database) ->
-    _Pid = spawn('kapi_maintenance', 'refresh_views', [Database]),
+    refresh_views(Database, kz_datamgr:db_classification(Database)).
+
+-spec refresh_views(kz_term:ne_binary(), kz_datamgr:db_classification() | kz_term:ne_binary()) -> 'ok'.
+refresh_views(_Database, 'undefined') ->
+    'ok';
+refresh_views(Database, 'modb') ->
+    kazoo_modb:refresh_views(Database),
+    _ = kazoo_bindings:map(binding({'refresh', Database}), [Database]),
+    'ok';
+refresh_views(Database, 'account') ->
+    refresh_account_db(Database);
+refresh_views(Database, _) ->
+    _ = kz_datamgr:refresh_views(Database),
+    _ = kazoo_bindings:map(binding({'refresh', Database}), [Database]),
+    'ok'.
+
+-spec refresh_account_db(kz_term:ne_binary()) -> 'ok'.
+refresh_account_db(Database) ->
+    AccountDb = kz_util:format_account_id(Database, 'encoded'),
+    AccountId = kz_util:format_account_id(Database, 'raw'),
+    _ = kz_datamgr:refresh_views(AccountDb),
+    kapps_account_config:migrate(AccountDb),
```

Comment:
why calling `migrate` here ?

Reply:
I moved this from `kapps_maint_listener` here. Should I moved to migrate function?

Reply:
yes, move to migrate